cmd/nerdctl/compose_images_linux_test.go

```diff
@@ -75,3 +76,80 @@ volumes:
 	base.ComposeCmd("-f", comp.YAMLFullPath(), "images").AssertOutContains(dbImageName)
 	base.ComposeCmd("-f", comp.YAMLFullPath(), "images").AssertOutContains(wordpressImageName)
 }
+
+func TestComposeImagesJson(t *testing.T) {
+	// `--format` is only supported in compose v2.
+	// Currently, CI is using compose v1.
```

Comment:
please update as above:
`--format is only supported in docker compose v2.`
`Currently, CI is using docker compose v1.`  

because we also support compose into nerdctl too

would you please update these [lines](https://github.com/containerd/nerdctl/pull/2157/files#diff-cf5e5f2df14711c5dd73f6508b0b5ce392bd61c21d06a05dd2f4b953d88f0d31R30) too

Reply:
Thanks. I updated the comments.